forest/src/cli/chain_cmd.rs

```diff
@@ -48,6 +69,81 @@ impl ChainCommands {
                 let cid: Cid = cid.parse().unwrap();
                 print_rpc_res_pretty(chain_get_block((CidJson(cid),)).await);
             }
+            Self::Export {
+                tipset,
+                recent_stateroots,
+                skip_old_messages,
+                output_path,
+            } => {
+                let recent_stateroots = match recent_stateroots {
+                    Some(rsrs) => {
+                        let rsrs = rsrs.to_owned();
+                        if rsrs < CHAIN_FINALITY {
+                            return cli_error_and_die(
+                                &format!(
+                                    "\recent-stateroots\" must be greater than {}",
+                                    CHAIN_FINALITY
+                                ),
+                                1,
+                            );
+                        }
+
+                        if rsrs == 0 && *skip_old_messages {
+                            return cli_error_and_die(
+                                "must pass recent stateroots along with skip-old-messages",
+                                1,
+                            );
+                        }
+
+                        rsrs
+                    }
+                    None => 0,
+                };
+
+                let chain_head = match chain_head().await {
+                    Ok(head) => head.0,
+                    Err(_) => return cli_error_and_die("Could not get network head", 1),
+                };
+
+                let output_path = match output_path {
+                    Some(path) => path.to_owned(),
+                    None => {
+                        let now = Utc::now();
+                        format!(
+                            "forest_snapshot_{}_{}_{}_{}.car",
```

Comment:
Could you add a short desc for the network and `high` before epoch to help readability?
E.g.
`forest_cali_snapshot_2021_07_14_high_73770`

Reply:
👍